src/styles/blocks/watches.scss

```diff
@@ -0,0 +1,46 @@
+.watches {
+  position: relative;
+  width: $watch-width;
+  height: $watch-height;
+  background-image: url(https://c0.klipartz.com/pngpicture/314/392/gratis-png-reloj-cara-reloj-digital-reloj-s-gratis.png);
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+  border-radius: 50%;
+
+  &__center {
+    position: absolute;
+    width: $watch-middle-width;
+    height: $watch-middle-height;
+    top: 50%;
+    left: 50%;
+    border-radius: 50%;
+    background-color: $item-bgc;
+    transform: translate(-50%, -50%);
+  }
+
+  &__minute-arrow {
+    @include similarItemValues;
```

Comment:
good that you group similar values in @mixin, but you are using it twice. In this case, it is useless. Just add one generic class like "watch__hand" and put these styles there

Reply:
` animation: rotate 60s linear infinite;` also you can put there

Reply:
As I understand acc BEM it is necessary to create new block, but I dont know is it correct, because I cannot to reuse block with this characteristics. I'll ask about it, thanks

Reply:
is there still an issue? now it looks correct according to BEM